Sources/Plasma/FeatureLib/pfJournalBook/pfJournalBook.cpp

```diff
@@ -1701,32 +1665,32 @@ void    pfJournalBook::ForceCacheCalculations()
 }
 
 // Tiny helper to convert hex values the *right* way
-static uint32_t   IConvertHex( const wchar_t *str )
+// TODO Is it safe to replace this with str.to_uint(16)?
```

Comment:
... probably...  Although this `IConvertHex` apparently completely ignores all characters that aren't `[0-9a-fA-F]`, which may cause a behavior difference if anything is relying on that (bad) behavior.